src/main/java/org/spongepowered/api/service/economy/EconomyService.java

```diff
@@ -69,49 +69,48 @@
     Set<Currency> getCurrencies();
 
     /**
-     * Gets the {@link UniqueAccount} with the specified {@link UUID}, if available.
+     * Returns whether a {@link UniqueAccount} exists with the specified {@link UUID}.
      *
-     * @param uuid The {@link UUID} of the account to get
-     * @return The {@link UniqueAccount}, if available.
+     * @param uuid The {@link UUID} of the account to check for
+     * @return Whether a {@link UniqueAccount} exists with the specified {@link UUID}
      */
-    Optional<UniqueAccount> getAccount(UUID uuid);
+    boolean hasAccount(UUID uuid);
 
     /**
-     * Gets the {@link Account} with the specified identifier, if available.
+     * Returns whether an {@link Account} with the specified identifier exists.
      *
-     * <p>The returned {@link Account} may be a {@link UniqueAccount} or
-     * a {@link VirtualAccount}, depending on the implementation.</p>
+     * <p>Depending on the implementation, the {@link Account} may be a {@link UniqueAccount} or
+     * a {@link VirtualAccount}.
      *
-     * @param identifier The identifier of the account to get
-     * @return The {@link VirtualAccount}, if available.
+     * @param identifier The identifier of the account to check for
+     * @return Whether an {@link Account} with the specified identifier exists
      */
-    Optional<Account> getAccount(String identifier);
-
+    boolean hasAccount(String identifier);
 
     /**
-     * Attempts to create a {@link UniqueAccount} for the user with the specified {@link UUID}.
+     * Gets the {@link UniqueAccount} for the user with the specified {@link UUID}.
      *
-     * <p>If an account already exists with the specified {@link UUID}, it will be
-     * returned. No further action will be taken.</p>
+     * <p>If an account does not already exists with the specified {@link UUID}, it will be
+     * created.</p>
      *
      * <p>Creation might fail if the provided {@link UUID} does not correspond to an actual
      * player, or for an implementation-defined reason.</p>
      *
-     * @param uuid The {@link UUID} of the account to create.
-     * @return The created {@link UniqueAccount}, if available.
+     * @param uuid The {@link UUID} of the account to get.
+     * @return The {@link UniqueAccount}, if available.
      */
-    Optional<UniqueAccount> createAccount(UUID uuid);
+    Optional<UniqueAccount> getAccount(UUID uuid);
```

Comment:
Personally, I prefer `getAccount` checking if an account exists, and if one does not exist, creating and returning that new account. This is what I did before the Economy API as it made more sense to me to create an account if one did not exist and return that newly created one. I do understand the confusion with the function naming though as it does go against what a getter is supposed to do, but I think naming it something else might make it even more confusing in my opinion, so I think it should stay named `getAccount`.

With what @kashike said, I do not see a time where having two separate functions (one to solely grab an account, and another to grab an account or create one if one does not exist) would come in handy. If the plugin creator wants to create a new account if one does not exist, they can go ahead and do so in the `getAccount` function. Otherwise, they can continue using it the way it currently works. But then again, it may not hurt to have both functions in there and leave it up to the discretion of the plugin creator on whether or not to use both of them, since there may be a chance that they would need both.

Regarding `hasAccount`, I think this should 100% be added without a doubt. This would clean up some code on my end and most likely for others.